### ISSUE TITLE
Email: Microsoft Comments to WCAG 3.0 FPWD (2)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#417]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0005.html): 

Scoring and Testing: Although the intent of the proposed scoring and rating system is to provide a more flexible conformance approach, it will make testing significantly more complex. Having five possible rating "bands" (a scale of 0 to 4) for each outcome will make all testing, and in particular manual testing, extremely complex. In addition, having different scoring methods (e.g., pass/fail, rating scales) and different percentages-as well as adjectival ratings-for the various rating scale "bands" for different criteria adds yet another layer of complexity.  We recommend that the proposed scoring and rating system be simplified to reduce the additional testing burden. For example, the scoring system should be more objective and data-driven, and avoid adjectival ratings which potentially introduce ambiguity and whose meaning may change when translated into other languages.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
